model/registry/gen-ai.yaml

```
@@ -85,3 +105,9 @@ groups:
         note: It's RECOMMENDED to format completions as JSON string matching [OpenAI messages format](https://platform.openai.com/docs/guides/text-generation)
         examples: ["[{'role': 'assistant', 'content': 'The capital of France is Paris.'}]"]
         tag: llm-generic-events
+      - id: operation.name
+        stability: experimental
+        type: string
+        brief: The name of the operation being performed.
+        examples: ['generate']
```

Maybe to be more aligned with known terms?
```suggestion
        examples: ['chat', 'completion']
```

---

nit: it seems OpenAI uses `chat.completion`, so I'd suggest to use it instead of `chat` unless I'm missing something

---

Yeah they do and it's highly confusing :( 
I'd use `chat` and `completion`. [Cohere](https://github.com/cohere-ai/cohere-python) is also using just `chat`